### ISSUE TITLE
Panic with hugo new with directory archetype in multilingual site

### ISSUE THREAD

**Issue by uidHUB** [gohugoio/hugo#9204]
### What version of Hugo are you using (`hugo version`)?

<pre>
hugo v0.89.4-AB01BA6E+extended linux/amd64 BuildDate=2021-11-17T08:24:09Z VendorInfo=gohugoio
</pre>

### Does this issue reproduce with the latest release?
```
$ hugo new posts/2021/11/12323

panic: [BUG] no Page found for "[...]content/ru/posts/2021/11/12323/index.md"

goroutine 1 [running]:
github.com/gohugoio/hugo/create.(*contentBuilder).applyArcheType(0xc00025c840, {0xc000888af0, 0x65}, {0xc000379b70, 0xe})
	/root/project/hugo/create/content.go:276 +0x354
github.com/gohugoio/hugo/create.(*contentBuilder).buildDir(0xc00025c840)
	/root/project/hugo/create/content.go:179 +0x7c5
github.com/gohugoio/hugo/create.NewContent.func1()
	/root/project/hugo/create/content.go:93 +0x154
github.com/gohugoio/hugo/create.NewContent(0xc000492fd0, {0x0, 0x0}, {0x7ffd9e3173e4, 0x66f045})
	/root/project/hugo/create/content.go:108 +0x5fe
github.com/gohugoio/hugo/commands.(*newCmd).newContent(0xc00048df80, 0xc0003d8f00, {0xc0003fc2b0, 0x1, 0x0})
	/root/project/hugo/commands/new.go:83 +0x150
github.com/spf13/cobra.(*Command).execute(0xc0003d8f00, {0xc0003fc220, 0x1, 0x1})
	/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0xc000584280)
	/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bc
github.com/gohugoio/hugo/commands.Execute({0xc00003c0a0, 0x2, 0x2})
	/root/project/hugo/commands/hugo.go:92 +0xb4
main.main()
	/root/project/hugo/main.go:23 +0x58
```

```
$ hugo new posts/2021/12

panic: [BUG] no Page found for "[...]/content/ru/posts/2021/12/index.md"

goroutine 1 [running]:
github.com/gohugoio/hugo/create.(*contentBuilder).applyArcheType(0xc000274600, {0xc000044b40, 0x5f}, {0xc00013a040, 0xe})
	/root/project/hugo/create/content.go:276 +0x354
github.com/gohugoio/hugo/create.(*contentBuilder).buildDir(0xc000274600)
	/root/project/hugo/create/content.go:179 +0x7c5
github.com/gohugoio/hugo/create.NewContent.func1()
	/root/project/hugo/create/content.go:93 +0x154
github.com/gohugoio/hugo/create.NewContent(0xc0000a9550, {0x0, 0x0}, {0x7ffde3c863ea, 0x66f045})
	/root/project/hugo/create/content.go:108 +0x5fe
github.com/gohugoio/hugo/commands.(*newCmd).newContent(0xc0002b1a10, 0xc00069db80, {0xc0002ede10, 0x1, 0x0})
	/root/project/hugo/commands/new.go:83 +0x150
github.com/spf13/cobra.(*Command).execute(0xc00069db80, {0xc0002edde0, 0x1, 0x1})
	/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0xc0003f0f00)
	/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bc
github.com/gohugoio/hugo/commands.Execute({0xc00011c070, 0x2, 0x2})
	/root/project/hugo/commands/hugo.go:92 +0xb4
main.main()
	/root/project/hugo/main.go:23 +0x58
```

BUT...

```
$ hugo new posts/test

Content dir "[...]/content/ru/posts/test" created
```

```
$ cat themes/ui-base/archetypes/posts/index.md
---
{{ $title := ((replace .Name "-" " ") | title) -}}
{{ $hash := (((printf "%s.%d.%s" ("6ba7b811-9dad-11d1-80b4-00c04fd430c8") (now.Unix) (delimit (shuffle (seq 999)) "")) | base64Encode) | sha1) -}}
{{ $uuid := (printf "%s-%s-5%s-%s%s-%s" (substr $hash 0 8) (substr $hash 8 4) (substr $hash 13 3) (index (slice "8" "9" "a" "b" | shuffle) 0) (substr $hash 17 3) (substr $hash 20 12)) -}}

title: '{{ $title }}'
description: ''

# --------------------------------------------------------------------------- #
# Meta.
# --------------------------------------------------------------------------- #

categories:
  - ''
tags:
  - ''

date: '{{ .Date }}'
hash: '{{ $hash }}'
uuid: '{{ $uuid }}'
slug: '{{ $uuid }}'

draft: 1
---
```

**Comment by Base-IQ-ly-Loud** [comment]
I think we may have mistook panic for irritation - which can and has proven to be able to run undetected without hindering performance.

Love.
EarthlyTIM

-------- Original Message --------

**Comment by KitsuneSolar** [comment]
The error goes away when I remove `theme = ['ui-base']` from `config.toml`

**Comment by KitsuneSolar** [comment]
Others ArcheTypes work well. Only **posts** doesn't work.

**Comment by KitsuneSolar** [comment]
Found a bug.

If there is a post in Russian content `content//ru/posts/2021/11/1636904963525` and there is a post in English content `content//en/posts/2021/11/1636904963525`, then a new post in Russian is not created.

BUT! If you delete an English post with a directory structure (`2021/11/1636904963525`), then everything starts working.

It is possible that an error will occur on the opposite condition.

**Comment by KitsuneSolar** [comment]
Checked on other sites. The error is repeated everywhere.
My configs...

**config.toml**

```
# ----------------------------------------------------------------------------------------------------------------------
# General.
# ----------------------------------------------------------------------------------------------------------------------

baseURL = 'http://***/'
theme = ['ui-base']
DefaultContentLanguage = 'ru'
defaultContentLanguageInSubdir = true
disableHugoGeneratorInject = true
paginate = 10
enableEmoji = true
pygmentsUseClasses = true
pygmentsCodeFences = true
publishdir = 'public'
enableRobotsTXT = true
rssLimit = 10
```

**languages.toml**

```
[ru]
title = 'iNFO Database'
description = ''
contentDir = 'content/ru'
languageName = 'Русский'
languageCode = 'ru-RU'
languageLang = 'ru'
weight = 10

[en]
title = 'iNFO Database'
description = ''
contentDir = 'content/en'
languageName = 'English'
languageCode = 'en-US'
languageLang = 'en'
weight = 20
```

**Comment by KitsuneSolar** [comment]
Tested on a clean installation.

New posts creating normal:

```
$ hugo new posts/2021/11/1637681669542
Content dir "[...]/content/ru/posts/2021/11/1637681669542" created

$ hugo new posts/2021/11/1637681679559
Content dir "[...]/content/ru/posts/2021/11/1637681679559" created

$ hugo new posts/2021/11/1637681680136
Content dir "[...]/content/ru/posts/2021/11/1637681680136" created
```

```
├── archetypes
│   └── posts
│       └── index.md
├── config
│   └── _default
│       ├── config.toml
│       └── languages.toml
├── content
│   └── ru
│       └── posts
│           └── 2021
│               └── 11
│                   ├── 1637681669542
│                   │   └── index.md
│                   ├── 1637681679559
│                   │   └── index.md
│                   └── 1637681680136
│                       └── index.md
```

BUT! Copy `content/ru` to `content/en`:

```
.
├── archetypes
│   └── posts
│       └── index.md
├── config
│   └── _default
│       ├── config.toml
│       └── languages.toml
├── content
│   ├── en
│   │   └── posts
│   │       └── 2021
│   │           └── 11
│   │               ├── 1637681669542
│   │               │   └── index.md
│   │               ├── 1637681679559
│   │               │   └── index.md
│   │               └── 1637681680136
│   │                   └── index.md
│   └── ru
│       └── posts
│           └── 2021
│               └── 11
│                   ├── 1637681669542
│                   │   └── index.md
│                   ├── 1637681679559
│                   │   └── index.md
│                   └── 1637681680136
│                       └── index.md
```

Create new post:

```
$hugo new posts/2021/11/test-post

panic: [BUG] no Page found for "[...]/content/ru/posts/2021/11/test-post/index.md"

goroutine 1 [running]:
github.com/gohugoio/hugo/create.(*contentBuilder).applyArcheType(0xc000276540, {0xc0008790a0, 0x65}, {0xc0004dac80, 0xe})
	/root/project/hugo/create/content.go:276 +0x354
github.com/gohugoio/hugo/create.(*contentBuilder).buildDir(0xc000276540)
	/root/project/hugo/create/content.go:179 +0x7c5
github.com/gohugoio/hugo/create.NewContent.func1()
	/root/project/hugo/create/content.go:93 +0x154
github.com/gohugoio/hugo/create.NewContent(0xc00055d550, {0x0, 0x0}, {0x7ffcb97d33ea, 0x66f045})
	/root/project/hugo/create/content.go:108 +0x5fe
github.com/gohugoio/hugo/commands.(*newCmd).newContent(0xc00044cc00, 0xc0000e1b80, {0xc0002ed9f0, 0x1, 0x0})
	/root/project/hugo/commands/new.go:83 +0x150
github.com/spf13/cobra.(*Command).execute(0xc0000e1b80, {0xc0002ed9c0, 0x1, 0x1})
	/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0xc0003f0c80)
	/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bc
github.com/gohugoio/hugo/commands.Execute({0xc00011c070, 0x2, 0x2})
	/root/project/hugo/commands/hugo.go:92 +0xb4
main.main()
	/root/project/hugo/main.go:23 +0x58
```

**Comment by KitsuneSolar** [comment]
I went back to version **0.88.1**
There is no error on this version.

**Comment by razonyang** [comment]
Got the same issue on hugo v0.89.4-AB01BA6E linux/amd64 BuildDate=2021-11-17T08:24:09Z VendorInfo=gohugoio

```text
hugo new --themesDir=../../ content/zh-cn/posts/first-post.md
panic: [BUG] no Page found for "/home/razonyang/Projects/razonyang/hugo-theme-bootstrap/exampleSite/content/zh-cn/posts/a.md"

goroutine 1 [running]:
github.com/gohugoio/hugo/create.(*contentBuilder).applyArcheType(0xc00066a840, {0xc0005f2c60, 0x5c}, {0xc0005c8920, 0xa})
        /root/project/hugo/create/content.go:276 +0x354
github.com/gohugoio/hugo/create.(*contentBuilder).buildFile(0xc00066a840)
        /root/project/hugo/create/content.go:244 +0x1ad
github.com/gohugoio/hugo/create.NewContent.func1()
        /root/project/hugo/create/content.go:104 +0x297
github.com/gohugoio/hugo/create.NewContent(0xc0006ef600, {0x0, 0x0}, {0x7ffe3d05f2bf, 0x5a3725})
        /root/project/hugo/create/content.go:108 +0x5fe
github.com/gohugoio/hugo/commands.(*newCmd).newContent(0xc0006dbfb0, 0xc0004fd400, {0xc00026ff60, 0x1, 0x0})
        /root/project/hugo/commands/new.go:83 +0x150
github.com/spf13/cobra.(*Command).execute(0xc0004fd400, {0xc00026ff20, 0x2, 0x2})
        /go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0xc0008b4780)
        /go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bc
github.com/gohugoio/hugo/commands.Execute({0xc00003a090, 0x3, 0x3})
        /root/project/hugo/commands/hugo.go:92 +0xb4
main.main()
        /root/project/hugo/main.go:23 +0x58
```

**Comment by jmooring** [comment]
This appears to be limited to directory archetypes in multilingual sites, triggered by the order in which you create content. The presence of a theme is irrelevant.

Simplified MRE

```bash
git clone --single-branch -b hugo-github-issue-9204 https://github.com/jmooring/hugo-testing hugo-github-issue-9204
cd hugo-github-issue-9204
hugo new content/en/post/test   # defaultContentLanguage
hugo new content/de/post/test
```

Now, clear the content directories, then create the content in reverse order.

```bash
rm -rf content/en/* content/de/*
hugo new content/de/post/test
hugo new content/en/post/test   # defaultContentLanguage
```

To avoid the error:
- If the defaultContentLanguage is `en`, create the `de` content first.
- If the defaultContentLanguage is `de`, create the `en` content first.

**Comment by KitsuneSolar** [comment]
@jmooring But at 0.88.1 there is no problem.

**Comment by jmooring** [comment]
@KitsuneSolar I know. That's why the issue is still open.

**Comment by KitsuneSolar** [comment]
@jmooring ok.

Yes, and it is stupid to somehow follow such an order. If `en` is the default language, then the first thing I do is create an article in the `en` language.

And only then I will translate into `de` language.

Sorry for my English. )

**Comment by jmooring** [comment]
@KitsuneSolar 

> Yes, and it is stupid to somehow follow such an order.

I know. That's why the issue is still open.

**Comment by KitsuneSolar** [comment]
0.91.0 did not fix bug :(

```
panic: [BUG] no Page found for "/[...]/content/ru/posts/2021/12/1640026896685_408365/index.md"

goroutine 1 [running]:
github.com/gohugoio/hugo/create.(*contentBuilder).applyArcheType(0xc000278600, {0xc000be0000, 0x73}, {0xc0009ca2e0, 0xe})
	/root/project/hugo/create/content.go:276 +0x354
github.com/gohugoio/hugo/create.(*contentBuilder).buildDir(0xc000278600)
	/root/project/hugo/create/content.go:179 +0x7c5
github.com/gohugoio/hugo/create.NewContent.func1()
	/root/project/hugo/create/content.go:93 +0x154
github.com/gohugoio/hugo/create.NewContent(0xc0000f0f20, {0x0, 0x0}, {0x7ffc4154931f, 0x66f0c5})
	/root/project/hugo/create/content.go:108 +0x5fe
github.com/gohugoio/hugo/commands.(*newCmd).newContent(0xc00011dd40, 0xc000472000, {0xc0003cedc0, 0x1, 0x0})
	/root/project/hugo/commands/new.go:83 +0x150
github.com/spf13/cobra.(*Command).execute(0xc000472000, {0xc0003ced90, 0x1, 0x1})
	/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0xc00091b180)
	/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bc
github.com/gohugoio/hugo/commands.Execute({0xc00003c0a0, 0x2, 0x2})
	/root/project/hugo/commands/hugo.go:92 +0xb4
main.main()
	/root/project/hugo/main.go:23 +0x58
```

**Comment by KitsuneSolar** [comment]
@jmooring 

> If the defaultContentLanguage is en, create the de content first.
> If the defaultContentLanguage is de, create the en content first.

This only works when there is no content in both directories. Once the directories are full, no new content can be created.

**Comment by viegasfh** [comment]
I am also having the same issue, and what I noticed is that it only happens in the content type subfolders. For instance, I have posts as my folder for contents of type post, and if I run:

```
hugo new -k post-bundle content/english/posts/new
```
This works without any issues. However, the minute I try to create a bundle in a folder inside posts, I get the error reported.

It happened to me a while ago, and I managed to get it to work, but unfortunately I cannot find the notes where I wrote down the procedure.

Has anyone found a workaround to this? The language sequence approach does not work for me.

**Comment by jmooring** [comment]
I ran into this today while looking at <https://discourse.gohugo.io/t/36792>.

Somewhat related: <https://github.com/gohugoio/hugo/issues/9196>

**Comment by Wirone** [comment]
Experiencing this issue on Hugo 0.96.0. I have simple `Makefile` target:

```
add-post: check-slug ## Add post inside proper directory
	$(eval POST_DIR=posts/`date +'%Y'`/`date +'%m'`)
	@hugo new "$(POST_DIR)/$(slug).md"
	@mkdir -p content/pl/$(POST_DIR)
	@cp "content/en/$(POST_DIR)/$(slug).md" "content/pl/$(POST_DIR)/$(slug).md"

check-slug:
ifndef slug
	$(error Slug not found, use slug=<slug>)
endif
```

and when running `make add-post slug="foo-bar"` I get:

```
panic: [BUG] no Page found for "/path/to/page/content/en/posts/2022/04/foo-bar.md"

goroutine 1 [running]:
github.com/gohugoio/hugo/create.(*contentBuilder).applyArcheType(0x14000321140, {0x140000538f0, 0x67}, {0x14000d1f310, 0x8})
        github.com/gohugoio/hugo/create/content.go:276 +0x238
github.com/gohugoio/hugo/create.(*contentBuilder).buildFile(0x14000321140)
        github.com/gohugoio/hugo/create/content.go:244 +0x184
github.com/gohugoio/hugo/create.NewContent.func1()
        github.com/gohugoio/hugo/create/content.go:104 +0x24c
github.com/gohugoio/hugo/create.NewContent(0x14000320a80, {0x0, 0x0}, {0x16fbd30d9, 0x36})
        github.com/gohugoio/hugo/create/content.go:108 +0x4ec
github.com/gohugoio/hugo/commands.(*newCmd).newContent(0x1400070f740, 0x140009e8780, {0x140004032f0, 0x1, 0x0?})
        github.com/gohugoio/hugo/commands/new.go:83 +0x164
github.com/spf13/cobra.(*Command).execute(0x140009e8780, {0x140004032c0, 0x1, 0x1})
        github.com/spf13/cobra@v1.4.0/command.go:856 +0x4c4
github.com/spf13/cobra.(*Command).ExecuteC(0x140009f0a00)
        github.com/spf13/cobra@v1.4.0/command.go:974 +0x354
github.com/gohugoio/hugo/commands.Execute({0x1400003c0d0, 0x2, 0x2})
        github.com/gohugoio/hugo/commands/hugo.go:93 +0x98
main.main()
        github.com/gohugoio/hugo/main.go:23 +0x54
make: *** [add-post] Error 2
```

Really weird, because I've used this target when I created it, being on the same version 😅

**Comment by Cristo-Conklin** [comment]
Just Use .md at the end solved it

` hugo new classy-cv
Error: failed to resolve "classy-cv" to a archetype template

hugo new portafolio/classy-cv.md
Content "D:\\programming\\apps\\hugo_0.98.0_Windows-64bit\\portafolio\\content\\portafolio\\classy-cv.md" created`

**Comment by varna9000** [comment]
Bump! This bug is still here in v.0.110.0
I confirm that the newest version without the bug is 0.88.1

**Comment by aakashnand** [comment]
I encountered this bug today. I cannot create post with default frontmatter in multilanguage.

**Comment by jmooring** [comment]
This is fixed in v0.123.0.
